doc/nrf/images/zigbee_signal_handler_09_leave.uml

```diff
@@ -0,0 +1,11 @@
+@startuml
+
+:ZB_ZDO_SIGNAL_LEAVE;
+
```

Comment:
It has been changed, now the coordinator forms network instead of calling the `start_network_rejoin()`.

This should look more like this:

> if (status == RET_OK) then (no)
else (yes)
        if (role == coordinator) then (no)
                :start_network_rejoin();
        else (yes)
                bdb_start_top_level_commissioning(ZB_BDB_NETWORK_FORMATION);
         endif
endif

Reply:
I cannot find that behavior inside the source code.

Reply:
The change was merged just yesterday, please check the most current master branch.

Reply:
And what was the reasoning behind this change? Can Zigbee coordinator leave the network it has formed?

Reply:
Yes it can, `zb_bdb_reset_via_local_action()` in the end indicates `leave` signal.